lib/metasploit/framework/data_service/remote/http/remote_loot_data_service.rb

```diff
@@ -28,7 +28,7 @@ def loot(opts = {})
   end
 
   def report_loot(opts)
-    self.post_data_async(LOOT_API_PATH, opts)
```

Comment:
Note to future reader: This was previously async, but it made it awkward to track IDs and updated file paths, and it also made tests non-deterministic as we couldn't assume the population of loot was actually persisted in the database before attempting to verify console output